byzcoin/bcadmin/main.go

```diff
@@ -21,6 +21,7 @@ import (
 	"github.com/qantik/qrgo"
 	"github.com/urfave/cli"
 	"go.dedis.ch/cothority/v3"
+	_ "go.dedis.ch/cothority/v3/bevm"
```

Comment:
Why is this in the PR ?

Reply:
Because BEVM is now part of the byzcoin-conodes, and there are BEVM instructions in the chain. So if BEVM is not included in `bcadmin`, the command to verify the chain, `bcadmin db replay`, fails...